plugins/modules/ios_snmp_server.py

```diff
@@ -20,7 +20,7 @@
   - This module provides declarative management of SNMP server on Cisco IOS devices.
 module: ios_snmp_server
 notes:
-  - Tested against Cisco IOSXE Version 17.3 on CML.
+  - Tested against Cisco IOSv Version <TODO> and IOS-XE v<TODO>.
```

Comment:
I have tested it against 17.3, we can change this back to `  - Tested against Cisco IOSXE Version 17.3 on CML.`